tests/testthat/test-epinow.R

```diff
@@ -1,13 +1,9 @@
 skip_on_cran()
 
-
-generation_time <- get_generation_time(disease = "SARS-CoV-2", source = "ganyani", max_value = 15)
-incubation_period <- get_incubation_period(disease = "SARS-CoV-2", source = "lauer", max_value = 15)
-reporting_delay <- dist_spec(
```

Comment:
I don't think this is an incredibly big deal but just nothing that all tests have now slightly changed as the reporting delay used now has no uncertainty.